index.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+  <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
+  <title>Crypto New World</title>
+</head>
+<body>
+  <main class="container">
+    <header>
+      <div class="mobile-header">
+        <nav class="nav-bar-mobile">
+          <div class="line line-one"></div>
+          <div class="line line-two"></div>
+          <div class="line line-three"></div>
+        </nav>
+        <div id="mobile-links">
+          <i class="fa-solid fa-x"></i>
+        <ul>
+          <li><a href="index.html"><img src="images/logo.png" alt=""></a></li>
+          <li><a href="index.html">Home</a></li>
+          <li class="list-margin"><a href="about.html">About</a></li>
+          <li class="list-margin"><a href="#">Program</a></li>
+          <li class="list-margin"><a href="#">Join</a></li>
+          <li class="list-margin"><a href="#">Sponsor</a></li>
+          <li class="list-margin"><a href="#">News</a></li>
+        </ul>
+      </div>
+    </div>
+      <div class="desktop-header">
+        <div class="top-menu">
+          <div class="menu-container">
+            <div class="icons-brand">
+              <i class="fa-brands fa-facebook-f"></i>
+              <i class="fa-brands fa-twitter"></i>    
+            </div>
+            <div class="sign">
+              <a>English</a>
+              <a>My Page</a>
+              <a>Logout</a>
```

Comment:
- [ ] Please wrap these links in a list, You can use `<ul>.` When we similar elements on the page It is a good practice to wrap them in the list and it will also make sure the code is semantic.